manager/controllers/mockup/policymanager.go

```diff
@@ -37,7 +38,8 @@ func (s *MockPolicyCompiler) GetPoliciesDecisions(in *pb.ApplicationContext) (*p
 		args := make(map[string]string)
 
 		var operationDecisions []*pb.OperationDecision
-		assetID := utils.GetAttribute("asset_id", dataset.GetDatasetId())
+		splittedID := strings.SplitN(dataset.GetDatasetId(), "/", 2)
+		assetID := splittedID[1]
```

Comment:
Safeguard in case datasetD is not of the form xxx/yyy ?

Reply:
Added